pybliometrics/scopus/scopus_search.py

```diff
@@ -229,7 +233,9 @@ def _join(item, key, sep=";", unescape=False):
     the elements are not None.
     """
     try:
-        return sep.join([get_unescaped_or_raw(d, key, unescape) or "" for d in item["affiliation"]])
+        if unescape:
+            return html_unescape(sep.join([d[key] or "" for d in item["affiliation"]]))
+        return sep.join([d[key] or "" for d in item["affiliation"]])
```

Comment:
This can be condensed, as the two lines are very similar. Something like:
```
string = sep.join([d[key] or "" for d in item["affiliation"]])
if unescape:
    return html_unescape(string)
else:
    return string
```